docs/wiki/framework/crew-served-weapons-framework.md

```diff
@@ -191,3 +191,29 @@ class CfgVehicles {
     };
 };
 ```
+
+## 3. Additional Configuration Parameters
+
+There are some additional config parameters available to extend functionality.
+
+### 3.1. `allowFireOnLoad`
+
+This option will make is so that whenever the weapon is reloaded, it will fire automatically. The intended use case of this is for mortars, where
```

Comment:
```suggestion
This option will make so that whenever the weapon is reloaded, it will fire automatically. The intended use case of this is for mortars, where
```